prettier/tests/__snapshots__/run_tests.js.snap

```diff
@@ -3838,16 +3854,26 @@ This is a simple dialog rendered by using a macro and a call block.
     num_seconds=5,
     version="1" %}
     {% widget_attribute "slides"
-      is_json=True %}[{"caption": "CRM Contacts App", "show_caption": true,
-      "link_url": "http://www.hubspot.com/crm", "alt_text": "Screenshot of CRM
-      Contacts", "img_src":
-      "http://go.hubspot.com/hubfs/Contacts-View-1.png?t=1430860504240",
-      "open_in_new_tab": true}, {"caption": "HubSpot CRM Contact Profile",
-      "show_caption": true, "link_url": "http://www.hubspot.com/", "alt_text":
-      "HubSpot CRM Contact Profile", "img_src":
-      "http://cdn2.hubspot.net/hubfs/53/Contact-Profile.png?t=1430860504240",
-      "open_in_new_tab": true}]
-  {% end_widget_attribute %}
+      is_json=True %}
+    [
+      {
+        "caption": "CRM Contacts App",
+        "show_caption": true,
+        "link_url": "http://www.hubspot.com/crm",
+        "alt_text": "Screenshot of CRM Contacts",
+        "img_src": "http://go.hubspot.com/hubfs/Contacts-View-1.png?t=1430860504240",
+        "open_in_new_tab": true
+      },
+      {
+        "caption": "HubSpot CRM Contact Profile",
+        "show_caption": true,
+        "link_url": "http://www.hubspot.com/",
+        "alt_text": "HubSpot CRM Contact Profile",
+        "img_src": "http://cdn2.hubspot.net/hubfs/53/Contact-Profile.png?t=1430860504240",
+        "open_in_new_tab": true
+      }
+    ]
+    {% end_widget_attribute %}
```

Comment:
The JSON is indented with the tag itself but not a level deeper. The prettier printer can indent but it's difficult to dedent the following closing tag because some of the leading whitespace comes from the HTML format step and the HubL printer just isn't aware of the current space.

I'm hoping that's reasonable?

Reply:
So the problem is that if we properly indent the JSON, the closing tag `{% end_widget_attribute %}` will be indented the same amount as the JSON instead of the opening tag?

Reply:
Good point, appreciate the push. After another look I realize that a similar issue had already been solved and we really just needed to `.trim()` the formatted JSON before adding it to the HubL doc. 

Before/after: 
![image](https://user-images.githubusercontent.com/9009552/200682007-23056d67-a7eb-43c5-828e-954dcd6a1f54.png)